src/base-styles/helperClasses/animations/transitions.css

```diff
@@ -0,0 +1,18 @@
+/*---
+title: Transitions
+name: transitions
+section: Animation Helpers
+---
+
+Adds standard transitions
+
+class          | description
+-------------- | ---------------------------------------------------------
+`trsn-default` | transitions all properties to default duration and timing
```

Comment:
Should use double hyphen `subject--verb` / `concept--modification` in name.

I know I gave a truncated example for the "transition" in this class name, but I'm wondering if we can trade off brevity for clarity in the name here:
- `trans--type`
- `transition--type`

I'd vote for spelling out `transition` for clarity. The time it takes to type the letters is trivial compared to the time it takes to understand what you're reading.

Reply:
I'm ok with `transition--type`